pom.xml

```diff
@@ -18,7 +18,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.adobe.commerce.cif</groupId>
     <artifactId>api-model</artifactId>
-    <version>1.2.0-SNAPSHOT</version>
+    <version>2.0.0-SNAPSHOT</version>
```

Comment:
Isn't this updated at next major release?

Reply:
I needed to update this to make the built working. Next major release will include this change (and maybe others)